bootstrap.sh

```diff
@@ -356,9 +356,17 @@ init_docker() {
     sudo_askpass /bin/chmod 544 /Library/PrivilegedHelperTools/com.docker.vmnetd
     sudo_askpass /bin/chmod 644 /Library/LaunchDaemons/com.docker.vmnetd.plist
     sudo_askpass /bin/launchctl load /Library/LaunchDaemons/com.docker.vmnetd.plist
+    logk
+  fi
+}
 
+start_docker() {
+  if ! docker system info &>/dev/null; then
```

Comment:
Do we want to do something for the else case?

Reply:
At that point it's running, thus, AFAIU we don't need to do anything else.